### DISCUSSION TITLE
Problems uploading files

### DISCUSSION THREAD

**Discussion by matthewdgregory** [prasathmani/tinyfilemanager#596] [Q&A]
Hello There,

I am using TFM to copy of legal programs to share with our users, but there not uploading, please help

![image](https://user-images.githubusercontent.com/48640307/125698371-cb3e9788-885b-4ba0-bdbd-de83d0ccc092.png)
`$allowed_upload_extensions = 'exe,iso,rar,zip';`

Thanks Matthew

**Comment by prasathmani** [comment]
check this out #441
